lib/remix.js

```diff
@@ -13,7 +13,7 @@ function injectMetadata(html, metadata) {
     metaTags += '<meta name="data-remix-' + key + '" content="' + metadata[key] + '">\n';
   });
 
-  return html.replace(/<head>\n/, '$&' + metaTags);
+  return html.replace(/<head>\n?/, '$&' + metaTags);
```

Comment:
I'd simplify this and just jam it right next to the closing `>`, forget about new line:

```js
return html.replace(/<head>/, '$&' + metaTags);
```

What about a page with no `<head>`?  This is legal html5

```html
<!DOCTYPE html>
<title>Messing with you</title>
Body Content
```

@Pomax might have ideas on how to do this for all cases.  I think it's fine to do a "we assume `<head>`" fix now.